README.md

```diff
@@ -6,12 +6,26 @@
 * kaustubhag@chromium.org
 
 ## Introduction
-The [SameSite](https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis-05#section-8.8) cookie attribute is designed to defend against CSRF attacks but currently does not take the scheme of the site into account. This was [originally to assist sites during their transition to https](https://github.com/w3c/webappsec-fetch-metadata/issues/34#issuecomment-527338651), however it results in the secure and insecure versions of the same host being considered same-site. A network attacker could thus impersonate http://site.example and use it to bypass SameSite protections on https://site.example. Between this security flaw and [HTTPS usage markedly increasing](https://transparencyreport.google.com/https/overview), we believe it is time to change this definition.
+The [SameSite](https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis-05#section-8.8) cookie attribute is designed to defend against CSRF attacks but currently does not take the scheme of the site into account. This was [originally to assist sites during their transition to https](https://github.com/w3c/webappsec-fetch-metadata/issues/34#issuecomment-527338651), however it results in the secure and insecure versions of the same host being considered same-site. A network attacker could thus impersonate http://site.example and use it to bypass `SameSite` protections on https://site.example. Between this security flaw and [HTTPS usage markedly increasing](https://transparencyreport.google.com/https/overview), we believe it is time to change this definition.
 
 The web ecosystem is already moving in this direction with the [HTML spec](https://html.spec.whatwg.org/multipage/origin.html#same-site) having been updated with the new definition of same site and the in-progress W3C PING [Privacy Threat Model](https://w3cping.github.io/privacy-threat-model/#terminology) utilizing it.
 
+### Example Attacks and Mitigations
+Below are two scenerios: the first one illustrates how the `SameSite` cookie attribute (specifically `Strict` in this example) can protect against CSRF, the second illustrates how "Schemeful Same-Site" enhances that protection.
+#### A Cross-Domain CSRF Attack
+An attacker sends you an email claiming you've won a sweepstakes. All you need to do if visit a website and click a button. You decide this feels legitimate and go visit the site https://mega-sweepstakes.example. The site has a big attractive button urging you to click it but, unbeknownst to you, the button doesn't claim your prize. Instead it links to your bank along with a query parameter telling your bank to transfer your money to the attacker's account: https://bank.example/transfer/custom?from=checking&to=evil-inc-slush-fund&amt=1000. 
+
+If your bank's cookies don't have the `SameSite=Strict` attribute then clicking that link will cause your browser to navigate to your bank's site, send along your login cookies, and cause you to be $1000 poorer.
```

Comment:
nit: the victim has to be signed in during the attack, such that the bank.com cookie indicates they are authorized to do that transaction.